nozzleconfig/nozzle_config_test.go

```diff
@@ -23,12 +23,18 @@ var _ = Describe("NozzleConfig", func() {
 		Expect(conf.DataDogAPIKey).To(Equal("<enter api key>"))
 		Expect(conf.DataDogTimeoutSeconds).To(BeEquivalentTo(5))
 		Expect(conf.FlushDurationSeconds).To(BeEquivalentTo(15))
+		Expect(conf.FlushMaxBytes).To(BeEquivalentTo(57671680))
 		Expect(conf.InsecureSSLSkipVerify).To(Equal(true))
```

Comment:
Note: added this because as I realized that I hadn't done it with https://github.com/DataDog/datadog-firehose-nozzle/pull/15 and it's a super small thing to add so I didn't think a separate PR was necessary